root/etc/confd/templates/www.conf.tmpl

```diff
@@ -217,7 +217,6 @@ catch_workers_output = yes
 ;                specified at startup with the -d argument
 ;php_admin_value[sendmail_path] = /usr/sbin/sendmail -t -i -f www@my.domain.com
 php_flag[display_errors] = off
-;php_admin_value[error_log] = /opt/rh/php55/root/var/log/php-fpm/www-error.log
```

Comment:
there is no reason we can't put that right into `/var/log/php`. Let's get this merged and I will review the various branches in `apache-php` and we can consolidate.